scripts/emit-feed-changelog-markdown.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+# Authors: Dylan Turner
+# Description:
+# - Convert JSON changelog data into a human-readable report
+# - Example:
+#     ./emit-feed-changelog-markdown.py ./docs/feed-changelog.json > ./docs/feed-changelog.md
+# - Note: Currently we only care about packages added/removed, but more info can be extracted from
+#   the JSON
+
+from argparse import ArgumentParser, Namespace
+from json import loads
+
+# The start text of the FEED_CHANGELOG.md file
+FEED_CHANGELOG_START: str = \
+    '# Changes to NI Linux RT Package Feeds\n\n' \
+        + '## Description\n\n' \
+        + 'This documents the full changes to the core/ and extra/ feeds for each version ' \
+        + 'release.\n\n' \
+        + 'The changes are too long to keep in the [Changelog](/CHANGELOG.md) itself, ' \
+        + 'so they are kept here\n\n' \
+        + 'Generate them by:\n    1. adding a line to ./docs/feed-changelog.json: ' \
+        + '`./scripts/diff-pkg-changes.py <branch of old version> <old version> >>' \
+        + '<branch of new version> <new version> >> ./docs/feed-changelog.json`\n    ' \
+        + '2. updating the markdown with `./scripts/emit-feed-changelog-markdown.py ' \
+        + 'feed-changelog.json > feed-changelog.md`\n\n'
```

Comment:
You can use triple-quote notation here to retain whitespace and avoid having to do the escape shenanigans.
```python3
FEED_CHANGELOG_START: str = """# Changes to NI Linux RT Package Feeds

## Description
This documents the full changes to the core/ and extra/ feeds for each version
...
"""
```